docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md

```diff
@@ -0,0 +1,42 @@
+# Add support for autoscaling filesystem for AWS
+
+Test verifies that for all runs with the file system autoscaling capability being enabled the daemon checks current filesystem usage and attach additional disks in case there is no enough space in the filesystem.
+
+**Prerequisites**:
+- Admin user
+
+| Steps | Actions | Expected results |
+|:-----:|--------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select the test tool | |
+| 4 | Click the ***Run with custom settings*** button | |
+| 5 | Expand the **Advanced** section | |
+| 6 | Select `On-demand` value for **Price type**. Tick _Do not mount storages_ option | |
+| 7 | Expand **Exec environment** section | |
+| 8 | Input `25` into the **_Disk (Gb)_** field | |
+| 9 | Click **Launch** button | |
+| 10 | At the Runs page, click the just-launched run | |
+| 11 | Wait until the SSH hyperlink appears | |
+| 12 | Click the SSH hyperlink | |
+| 13 | In the opened tab, enter and perform the command `df -hT` | File system space usage table is shown in the response |
+| 14 | Store ***Avail*** value for filesystem with type `btrfs` | |
+| 15 | Enter and perform the command <br> `fallocate -l <file_size>G test.big`, <br> where `<file_size>` is a value equals to the value kept at the step 14 minus 1 | 
+| 16 | Return to the run log page | |
+| 17 | Wait until the ***FilesystemAutoscaling*** task appears | |
+| 18 | Click ***FilesystemAutoscaling*** task | Log contains record <br> `Filesystem /ebs was autoscaled <value1>G + <value2>G = <value3>G.`, <br> where <br> `<value1>` is value kept in the step 14, <br> `<value2>` is `<value1> * 0.5`, <br> `<value3> = <value1> + <value2>` |
```

Comment:
`<value1>` in this step shall be equal to the **Size** value for filesystem with type `btrfs` in output of step 13 (not **Avail** value)